PilotDataPlatform/metadata
# PILOT-1292 Use init container for running migrations

## Summary

- Change Docker files to run Alembic migrations in their own container / build stage.
- Move local default Postgres credentials from `docker-compose.yaml` to `config.py`.
- Update readme with new template and migration instructions.

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1292

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

- Run the service locally and ensure that migrations run. They are running in Actions.
- Compare to the [project service](https://github.com/PilotDataPlatform/project) for consistency of approach.
